packages/abi-typegen/package.json

```diff
@@ -43,10 +43,9 @@
     "dist"
   ],
   "scripts": {
-    "pretest": "run-s build:forc:debug build:forc:release",
+    "pretest": "pnpm build:forc",
     "build": "tsup",
-    "build:forc:debug": "pnpm fuels-forc build -p test/fixtures/forc-projects",
-    "build:forc:release": "pnpm fuels-forc build -p test/fixtures/forc-projects --release",
+    "build:forc": "pnpm fuels-forc build -p test/fixtures/forc-projects --release",
```

Comment:
What's the reason for removing the `debug` build? I assume this was testing support for both debug and release?

Reply:
Because `--json-abi-with-callpaths` is now on by default and the `debug` is now unnecessary because it was used only to test compatibility with that flag.